.env.example

```
@@ -465,3 +465,14 @@ DEBUG=false
 WANDB_SILENT=false
+
+# ============================================
+# Optional: Linear Integration
+# ============================================
+# Create issues from game feedback in Linear
+# Get API key from Linear > Settings > Account > Security & Access > API
+LINEAR_API_KEY=
+# Babylon team ID in Linear
+LINEAR_TEAM_ID=dbaea5df-7f3b-4747-b260-4aa5f6270228
+# Game Feedback label ID in Linear (optional - issues created without label if missing)
+LINEAR_GAME_FEEDBACK_LABEL_ID=44492495-2a60-4852-8525-b351a93087c5
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LINEAR_GAME_FEEDBACK_LABEL_ID key should go before the LINEAR_TEAM_ID key

(UnorderedKey)